spec.html

```diff
@@ -5876,7 +5869,7 @@ <h1>CreateArrayFromList ( _elements_ )</h1>
 
     <emu-clause id="sec-lengthofarraylike" aoid="LengthOfArrayLike">
       <h1>LengthOfArrayLike ( _obj_ )</h1>
-      <p>The abstract operation LengthOfArrayLike returns the value of the *"length"* property of an array-like object.</p>
+      <p>The abstract operation LengthOfArrayLike takes argument _obj_. It returns the value of the *"length"* property of an array-like object. It performs the following steps:</p>
```

Comment:
```suggestion
      <p>The abstract operation LengthOfArrayLike takes argument _obj_ (an array-like object). It returns the value of the *"length"* property of _obj_. It performs the following steps:</p>
```

Reply:
There's no actual requirement that _obj_ be array-like, contrary to what the existing prose says, so I don't want to add this one.

Reply:
There are abstract ops where we say "(an Object)" but then in the algorithm steps we throw if it's observed not to be an object. What's the difference here?

Reply:
We should get rid of those too. I got rid of several as I noticed them.

Reply:
Okay are you going to get rid of them in this PR?

Reply:
Sure.